CMakeLists.txt

```diff
@@ -856,8 +870,12 @@ if (BUILD_TESTS)
       TEST token_bucket_test SOURCES TokenBucketTest.cpp
       TEST traits_test SOURCES TraitsTest.cpp
       TEST try_test SOURCES TryTest.cpp
+      TEST type_list_test SOURCES TypeListTest.cpp
```

Comment:
This needs to be `WINDOWS_DISABLED`, as it doesn't currently work under MSVC due to compiler issues.

Reply:
Just marked it `WINDOWS_DISABLED`. Are there other tests that I've enabled in the CMake build which break Windows? I don't readily have access to a machine where I can build folly with MSVC 2017 to test.